docs/Alica_Base.md

```diff
@@ -35,18 +33,13 @@ namespace msl
 		cc = new alica::ConditionCreator();
 		uc = new alica::UtilityFunctionCreator();
 		crc = new alica::ConstraintCreator();
-
 		
-		//You can write your own clock and communication you only need to inherit from the IAlicaClock or IAlicaCommunication
-		//ROS Communicator and ROSCLOCK 
+		//ROS Communicator
```

Comment:
But I can still supply my own clock. Especially if i want to interface with ros.